### DISCUSSION TITLE
lock non-connected users

### DISCUSSION THREAD

**Discussion by veranderson** [gravitational/teleport#36491] [Q&A]
We would like to lock login by non-connected users.
I would like to lock a user who has not logged in for 30 days. Is this possible through rule or user settings?

If there is another way, please advise.  :-)

**Comment by stevenGravy** [comment]
Teleport does not enforce that.   You can lock users and use the audit log to get the last usage for a user.

**Reply by veranderson** [reply]
Thank you for your answer.  :-)

**Comment by veranderson** [comment]
Thank you for your response to our inquiry.
